jablib/src/main/java/org/jabref/logic/bibtex/BibtexStandard.java

```diff
@@ -0,0 +1,11 @@
+package org.jabref.logic.bibtex;
+
+import java.util.Arrays;
+import java.util.List;
+
+/// Contains various information or constants about the BibTeX standard.
+public class BibtexStandard {
+    /// Source of disallowed characters: <https://tex.stackexchange.com/a/408548/9075>
+    /// These characters are disallowed in BibTeX keys.
+    public static final List<Character> DISALLOWED_CHARACTERS = Arrays.asList('{', '}', '(', ')', ',', '=', '\\', '"', '#', '%', '~', '\'');
```

Comment:
We already have them
 https://github.com/JabRef/jabref/blob/861b00b05babeee88576623a33b7546825c6ff72/jablib/src/main/java/org/jabref/logic/citationkeypattern/CitationKeyGenerator.java#L34-L39